src/app/scripts/cac/pages/cac-pages-home.js

```diff
@@ -201,7 +214,20 @@ CAC.Pages.Home = (function ($, ModeOptions,  MapControl, TripOptions, SearchPara
         }
 
         // load directions if origin and destination set
-        $(document).ready(directionsControl.setFromUserPreferences());
+        $(document).ready(loadInitialMethod);
+    }
+
+    function loadInitialMethod() {
+        if (!UserPreferences.isDefault('method')) {
+            var method = UserPreferences.getPreference('method');
```

Comment:
Just noting for future reference: I suspect we can ditch the `method` user preference altogether, since it's not state that survives refresh.